code/src/index.css

```diff
@@ -5,9 +5,159 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgba(0, 0, 0, 0.89);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.container {
+  width: 320px;
+  margin: 0 auto;
+  margin-bottom: 40px;
+}
+
+.header {
+  border-bottom: rgb(54, 54, 54) 1px solid;
+  margin-top: 15px;
+}
+
+.heading {
+  color: #ffffff;
+  font-size: 16px;
+  font-family: Helvetica, sans-serif;
+  margin-bottom: 2px;
+}
+
+.icon-container {
+  display: none;
+  position: absolute;
+  justify-content: space-evenly;
+  padding: 40% 0;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+}
+
+.album-container:hover .icon-container {
+  display: flex;
+}
+
+.icon {
+  filter: invert();
+}
+
+.play {
+  width: 50px;
+  padding: 0 20px;
+  /* filter: invert(1); */
+}
+.dots {
+  width: 20px;
+  /* filter: invert(1); */
+}
+.heart {
+  width: 25px;
+  /* filter: invert(1); */
+}
+
+.play:hover {
+  width: 70px;
+  transition: 0.3s;
+  padding: 0 10px;
+  opacity: 1;
+}
+
+.album-container {
+  width: 300px;
+  height: 300px;
+}
+
+.overlay {
+  position: absolute;
+  width: 300px;
+  height: 300px;
+}
+
+.album-container:hover .overlay {
+  display: flex;
+  background: rgba(0, 0, 0, 0.6);
+  transition: 0.5s;
+}
+
+.article-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  position: relative;
+  grid-gap: 10px;
+  padding-top: 10px;
+  margin-top: 15px;
+}
+
+.article-unit {
+  display: flex;
+  flex-direction: column;
+}
+
+.article {
+  margin-top: 5px;
+  width: 250px;
+}
+
+.album-image {
+  width: 300px;
+}
+
+.song-name {
+  color: #ffffff;
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+  margin-bottom: 0;
+}
+
+.song-name:hover {
+  text-decoration: underline;
+}
+
+.artist-name {
+  color: #a0a0a0;
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+  margin-top: 0.2cm;
+}
+
+.artist-name:hover {
+  text-decoration: underline;
+}
+
+.link {
+  text-decoration: none;
+}
+
+/* tablet menu */
+
+@media (min-width: 768px) {
+  .article-container {
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 40px;
+  }
+
+  .container {
+    width: 600px;
+  }
+}
+
+/* desktop menu */
+
+@media (min-width: 992px) {
+  .article-container {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+
+  .container {
+    width: 1230px;
```

Comment:
This width is bigger than the min-width, so screens with a size between 992-1230 wont see all the content.